sky/templates/kubernetes-ray.yml.j2

```diff
@@ -96,7 +99,10 @@ provider:
             - protocol: TCP
               port: 22
               targetPort: 22
+      {% endif %}
+      {% if num_nodes > 1 %}
```

Comment:
Just curious, why is the following service ever required? It seems 8265 or 10001 does not need to be exposed to public.